torch/special/__init__.py

```diff
@@ -820,3 +862,135 @@
     tensor([1., 1., 1.])
 
 """.format(**common_args))
+
+jacobi_elliptic_k_cd = _add_docstr(_special.special_jacobi_elliptic_k_cd,
+                                   r"""
+Jacobi elliptic function :math:`cd`.
```

Comment:
These descriptions are totally fine for now (because the priority should be implementation correctness), but we may want to elaborate on these operations in the future

Reply:
Aha! One tweak: start the docs with the schema string, like in the above:

```
gammainc(input, other, *, out=None) -> Tensor
```

Reply:
Aha! One tweak: start the docs with the schema string, like in the above:

```
gammainc(input, other, *, out=None) -> Tensor
```

Reply:
No problem! Updated!